.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,49 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
+{
```

Comment:
Do not merge this until pointing to main

Reply:
Merging this into the `multi-scan-time` PR first could be helpful because it fixes some docstrings/release notes in that PR that were misformatted and thus caused the CI to fail

Reply:
But waiting until it points at main is also fine; either works

Reply:
That would make it more difficult to review the other PR, I'd rather merge it with a few actions failing, knowing that all actions will pass on this PR before merging this one.

Reply:
Works for me! Whatever's easiest to review :)